plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/MultipathSCSIAdapterBase.java

```diff
@@ -366,60 +360,42 @@ public KVMPhysicalDisk copyPhysicalDisk(KVMPhysicalDisk disk, String name, KVMSt
         LOGGER.info("Copying TO destination physical disk " + destDisk.getPath() + ", size: " + destDisk.getSize() + ", virtualsize: " + destDisk.getVirtualSize()+ ", format: " + destDisk.getFormat());
         QemuImgFile srcFile = new QemuImgFile(disk.getPath(), disk.getFormat());
         QemuImgFile destFile = new QemuImgFile(destDisk.getPath(), destDisk.getFormat());
-        LOGGER.debug("Starting COPY from source downloaded template " + srcFile.getFileName() + " to Primera volume: " + destDisk.getPath());
+
+        LOGGER.debug("Starting COPY from source path " + srcFile.getFileName() + " to target volume path: " + destDisk.getPath());
+
         ScriptResult result = runScript(copyScript, timeout, destDisk.getFormat().toString().toLowerCase(), srcFile.getFileName(), destFile.getFileName());
+        /**Script script = new Script(
+            String.format("%s %s %s %s", copyScript, destDisk.getFormat().toString().toLowerCase(), srcFile.getFileName(), destFile.getFileName()),
+            Duration.millis(timeout),
+            LOGGER);
+
+        script.execute();
+        int rc = script.getExitValue();
+        */
```

Comment:
remove this comment if not needed

Reply:
addressed in commit 42780e76163f72e7663b78945e74e62325fb4158 and 018e55678d494bdb53f1385fba57bd84031ae4a4